pkgs/tools/system/bashtop/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, fetchFromGitHub
+, bash_5
+, curl
+, lm_sensors
+, procps-ng
```

```suggestion
, procps
```

`procps-ng` is an alias to `procps`